platform/app/.recipes/Nginx-Orthanc/config/nginx.conf

```diff
@@ -57,6 +57,7 @@ http {
       # By default, this endpoint is protected by CORS (cross-origin-resource-sharing)
       # You can add headers to allow other domains to request this resource.
       # See the "Updating CORS Settings" example below
+      add_header 'Access-Control-Allow-Origin' '*' always;
```

Comment:
can you add this to orthanc keycloak as well?

Reply:
It already exists in the orthanc keycloak instance - only this one was needed.